src/main/java/uk/ac/cam/cl/dtg/segue/api/SegueContentFacade.java

```diff
@@ -54,8 +54,8 @@
 /**
  * Segue Content Facade
  * <br>
- * This class specifically caters for the Rutherford physics server and is expected to provide extended functionality to
- * the Segue api for use only on the Rutherford site.
+ * This class specifically caters for the Rutherford Computer Science server and is expected to provide extended
+ * functionality to the Segue api for use only on the Rutherford site.
```

Comment:
Given we're not running on UoC's servers, the name may need updating? Then again, I think the database is still referred to in the code as Rutherford in some places so I'm not sure...